Resources/Prototypes/silicon-laws.yml

```diff
@@ -170,5 +507,15 @@
     Crewsimov: 0.25
     Corporate: 1
     NTDefault: 1
+    CommandmentsLawset: 1
+    PaladinLawset: 1
+    LiveLetLiveLaws: 1
+    EfficiencyLawset: 1
+    RobocopLawset: 1
+    OverlordLawset: 0.5
+    DungeonMasterLawset: 0.5
+    PainterLawset: 1
+    AntimovLawset: 0.5
```

Comment:
bump it down to 0.25 or 0.1

Reply:
To all of them or only overlord dm painter and antimov?